### ISSUE TITLE
Using two processes - teams and lync - compete against each other

### ISSUE THREAD

**Issue by psyciknz** [andrew-schofield/openhab-video-call-updater#1]
So have it set for teams = 2 and lync = 0.

Found that when it runs, it evaludates teams at 10 (while on a call) then when it evaluates lync it turns the object off....
I might be different to use in that I have one switch item for all functions.

**Comment by andrew-schofield** [comment]
Yeah, I designed it for the use case of a switch item per application, simply because I need to run this on multiple machines at once.

In OpenHAB I've got multiple items grouped together with a "One ON then ON else OFF" aggregation so that if any application is running on any machine, it will trigger the "On a Video Call" state.

**Comment by psyciknz** [comment]
Ahh makes sense....So I have hacked it to break out of a loop if one of them are running....took me a while powershell is a very different beast for return values etc.

Or I could have made multiple items like you and put them in a group switch I guess.
